.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "3.6"
+
+install:
+  - "pip install pytest"
+  - "pip install -r requirements.txt"
+  - "pip install coveralls"
+
+script: pytest --cov=app
```

Comment:
Since we do not have tests yet, remove this line and everything after it.

This will be implemented when we start writing tests. I will not be merging anything with failing CI checks.

Reply:
ok